grabbit/core.py

```diff
@@ -193,6 +331,10 @@ def __init__(self, path, config=None, index=None, dynamic_getters=False,
         else:
             self.load_index(index)
 
+        if config and 'default_path_patterns' in config:
```

Comment:
Maybe store the defaults in `self.path_patterns` (can be concatenated with value passed in init arg if needed), and then pass the whole thing to the `File` init later? Unless there's a reason we can't do it all in one shot, I'm not crazy about setting `File` attributes manually post-initialization.